parlai/crowdsourcing/tasks/turn_annotations/frontend/components/onboarding_components.jsx

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+import React from "react";
+import { ErrorBoundary } from './error_boundary.jsx';
+import { Checkboxes } from './checkboxes.jsx';
+const ONBOARDING_MIN_CORRECT = 4;
+const ONBOARDING_MAX_INCORRECT = 3;
+const ONBOARDING_MAX_FAILURES_ALLOWED = 1;
+var onboardingFailuresCount = 0;
+
+var renderOnboardingFail = function () {
+    // Update the UI
+    document.getElementById("onboarding-submit-button").style.display = 'none';
+
+    alert('Sorry, you\'ve exceeded the maximum amount of tries to label the sample conversation correctly, and thus we don\'t believe you can complete the task correctly. Please return the HIT.')
+}
+
+function arraysEqual(_arr1, _arr2) {
```

Comment:
Do we need to have this function inside this individual task? Boilerplate again

Reply:
This could be in a utility class somewhere, but it's not something that's really Mephisto specific.